templates/index.html

```diff
@@ -49,11 +49,11 @@
           <div class="col-6 col-start-large-4">
             <div class="p-card--outline">
               <img src="https://assets.ubuntu.com/v1/6e94860c-menu-icon--product-juju.svg" alt="" style="display: block; width: 50px;">
-              <h1 class="p-heading--three">The Python Operator Framework</h1>
-              <p class="p-heading--four">The Python Operator Framework provides a simple, lightweight, and powerful way of writing Juju charms, the best way to encapsulate operational experience in code.</p>
+              <h1 class="p-heading--three">The Charmed Operator Framework</h1>
+              <p class="p-heading--four">The Charmed Operator Framework provides a simple, lightweight, and powerful way of writing charmed operators, the best way to encapsulate operational experience in code.</p>
               <ul class="p-inline-list u-sv3">
-                <li class="p-inline-list__item"><a href="https://github.com/canonical/operator/" class="p-button--neutral">Get involved</a></li>
-                <li class="p-inline-list__item"><a href="https://juju.is/contact-us" class="p-button--outline js-invoke-modal">Contact us</a></li>
+                <li class="p-inline-list__item"><a href="https://juju.is/" class="p-button--neutral">What is the Charmed Operator Framework?</a></li>
```

Comment:
```suggestion
                <li class="p-inline-list__item"><a href="https://juju.is/" class="p-button--neutral">Learn more</a></li>
```

It will make it a shorter.